python/tests/res/fm/test_util.py

```diff
@@ -0,0 +1,73 @@
+
+from ecl.util.test import TestAreaContext
+from tests import ResTest
+from res.fm.util import *
+
+import subprocess
+import os
+import sys
+
+class UtilTest(ResTest):
+    def test_template_parser(self):
+        with TestAreaContext("util/template") as tac:
+            with open("template", 'w') as template_file:
+                template_file.write(
+                    "FILENAME\n"+
+                    "F1 {{parameters.key1.subkey1}}\n"+
+                    "FX {{parameters.key1.subkey2}}\n"+
+                    "F1_T {{parameters.key2.subkey1}}")
+
+            with open("parameters.json", 'w') as json_file:
+                json_file.write('{\n'+\
+                    '   "key1": {\n'+\
+                    '       "subkey1": 1999.22,\n'+\
+                    '       "subkey2": 200 \n' +\
+                    '   },\n'+\
+                    '   "key2": {\n'+\
+                    '       "subkey1": 300\n'+\
+                    '   },'+\
+                    '}')
+
+            render_template('parameters.json','template','parameter_file')
+
+            with open("parameter_file", 'r') as parameter_file:
+                expected_output = "FILENAME\n"+\
+                    "F1 1999.22\n"+\
+                    "FX 200\n"+\
+                    "F1_T 300"
+
+                self.assertEqual(parameter_file.read(), expected_output)
+
+    def test_template_executable(self):
+        with TestAreaContext("util/template") as tac:
+            with open("template", 'w') as template_file:
+                template_file.write(
+                    "FILENAME\n"+
+                    "F1 {{parameters.key1.subkey1}}\n"+
+                    "FX {{parameters.key1.subkey2}}\n"+
+                    "F1_T {{parameters.key2.subkey1}}")
+
+            with open("parameters.json", 'w') as json_file:
+                json_file.write('{\n'+ \
```

Comment:
I think this would be more readable if specified as a `dict` and then dumped with the `json` library..